pkg/jobs/resultcols.go

```diff
@@ -21,6 +21,13 @@ var BulkJobExecutionResultHeader = colinfo.ResultColumns{
 	{Name: "bytes", Typ: types.Int},
 }
 
+var BackupRestoreJobResultHeader = colinfo.ResultColumns{
+	{Name: "job_id", Typ: types.Int},
+	{Name: "status", Typ: types.String},
+	{Name: "fraction_completed", Typ: types.Float},
+	{Name: "rows", Typ: types.Int},
+}
+
```

Comment:
As per [this discussion](https://cockroachlabs.slack.com/archives/C01RX2G8LT1/p1730999089400029), `bytes` is still considered an important column for `IMPORT` jobs. May want to create a separate PR to remove the `index_rows` column though, although not too important as of right now,